replays/build

```diff
@@ -8,32 +8,50 @@
 const fs = require('fs');
 const crypto = require('crypto');
 
+let configBuf = fs.readFileSync('config/config.js', {encoding: 'utf8'});
+const context = {Config: {}};
+require('vm').runInNewContext(configBuf, context);
+const Config = context.Config;
+
 process.chdir(__dirname);
 
 function updateIndex() {
-	let indexContents = fs.readFileSync('theme/wrapper.inc.template.php', {encoding: 'utf8'});
-
-	// add hashes to js and css files
-	process.stdout.write("Updating hashes... ");
-	indexContents = indexContents.replace(/(src|href)="\/(.*?)\?[a-z0-9]*?"/g, function (a, b, c) {
-		let hash = Math.random(); // just in case creating the hash fails
-		try {
-			var filepath = c;
-			if (c.includes('/play.pokemonshowdown.com/')) {
-				const filename = c.replace('/play.pokemonshowdown.com/', '');
-				filepath = '../' + filename;
-			}
-			const fstr = fs.readFileSync(filepath, {encoding: 'utf8'});
-			hash = crypto.createHash('md5').update(fstr).digest('hex').substr(0, 8);
-		} catch (e) {}
-
-		return b + '="/' + c + '?' + hash + '"';
-	});
-	console.log("DONE");
-
-	process.stdout.write("Writing new `wrapper.inc.php` file... ");
-	fs.writeFileSync('theme/wrapper.inc.php', indexContents);
-	console.log("DONE");
+        let indexContents = fs.readFileSync('theme/wrapper.inc.template.php', {encoding: 'utf8'});
+        let battleContents = fs.readFileSync('battle.template.php', {encoding: 'utf8'});
+
+        // add hashes to js and css files
+        process.stdout.write("Updating hashes... ");
+        indexContents = indexContents.replace(/(src|href)="\/(.*?)(\?[a-z0-9]*?)?"/g, runReplace);
+        battleContents = battleContents.replace(/(src|href)="\/(.*?)(\?[a-z0-9]*?)?"/g, runReplace);
+        console.log("DONE");
+
+        process.stdout.write("Writing new `wrapper.inc.php` file... ");
+        fs.writeFileSync('theme/wrapper.inc.php', indexContents);
+        console.log("DONE");
+        process.stdout.write("Writing new `battle.php` file... ");
+        fs.writeFileSync('battle.php', battleContents);
+        console.log("DONE");
+
+}
+
+function runReplace(a, b, c) {
```

Comment:
I actually think this function is less readable out-of-line rather than in-line where it happens... It also makes the diff unreadable, although the reindentation isn't doing it any favors...

Reply:
`runReplace` is called twice, both on `indexContents` and `battleContents`, and I don't want to repeat that twice.

Reply:
Oh, I see. Can you fix the reindentation, though? This file should be tab-indented.

Reply:
OK.